### PR TITLE
CheckCamelCase and CheckPascalCase method added to String category

### PR DESCRIPTION
**Pull request by suryapratapsinghsuryavanshi**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new JavaScript files are placed inside an existing directory.
* [x] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: String/CheckCamelCase.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * CheckCamelCase method returns true if the string in camelCase, else return the false.
+ * @param {String} varName the name of the variable to check.
+ * @returns `Boolean` return true if the string is in camelCase, else return false.
+ */
+const CheckCamelCase = (varName) => {
+  // firstly, check that input is a string or not.
+  if (typeof varName !== 'string') {
+    return 'Not string(s)'
```
Comment (raklaptudirm) [inline String/CheckCamelCase.js:13]:
```suggestion
    return new TypeError("Argument is not a string.")
```

File: String/CheckPascalCase.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * CheckPascalCase method returns true if the string in PascalCase, else return the false.
+ * @param {String} VarName the name of the variable to check.
+ * @returns `Boolean` return true if the string is in PascalCase, else return false.
+ */
+const CheckPascalCase = (VarName) => {
+  // firstly, check that input is a string or not.
+  if (typeof VarName !== 'string') {
+    return 'Not string(s)'
```
Comment (raklaptudirm) [inline String/CheckPascalCase.js:13]:
```suggestion
    return new TypeError("Argument is not a string.")
```

**Review by raklaptudirm — APPROVED** [review]

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Please fix the style issues.

**Review by raklaptudirm — APPROVED** [review]
I did forget `standard.js` requires single quotes in it's style guide.

**Comment by suryapratapsinghsuryavanshi** [comment]
It's okay, there is a lot of error in making big things. :)

**Pull request merged**